internal/conduit/models/models.go

```diff
@@ -0,0 +1,28 @@
+package models
+
+import (
+	"time"
+)
+
+type RegisterType string
+
+type TransitData struct {
+	Timestamp time.Time
+
+	Type  RegisterType
+	Value interface{}
+}
+
+type TransitChannel = chan TransitData
+
+func NewTransitChannel() TransitChannel {
+	return make(chan TransitData)
+}
+
+type ComponentType int
+
+const (
+	Oracle   ComponentType = 0
+	Pipe     ComponentType = 1
+	Conveyor ComponentType = 2
+)
```

Comment:
The constants probably better put in a different file ? More constants are to come as this progresses.

Reply:
[1e84ea9](https://github.com/base-org/pessimism/pull/3/commits/1e84ea94c6a1408b28cb88974fa6ab6332315ce5)